tests/common/factories/group.py

```diff
@@ -45,7 +45,7 @@ class OpenGroup(Group):
 
 
 class RestrictedGroup(Group):
-    name = factory.Sequence(lambda n: "Test Restricted Group {n}".format(n=str(n)))
+    name = factory.Sequence(lambda n: "Test Restricted Grp {n}".format(n=str(n)))
```

Comment:
I couldn't think of a better way to shorten this string by 1-2 characters. I'm all ears if anyone has a better idea.

Reply:
`"Restricted Group {n}"`?